src/test/java/jalau/at18/katas/bankocr/adriana/CharacterTypeTest.java

```diff
@@ -0,0 +1,18 @@
+package jalau.at18.katas.bankocr.adriana;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class CharacterTypeTest {
+
+     @Test
+     public void shouldShowItsImage() {
```

Comment:
Baby step... good

Reply:
Yes, that was my first baby step...but I should delete it, because I don't use it at the kata solution...